test/Altinn.Profile.Tests/IntegrationTests/TelemetryTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Altinn.Profile.Integrations.OrganizationNotificationAddressRegistry;
+using Altinn.Profile.Integrations.OrganizationNotificationAddressRegistry.Models;
+using Altinn.Profile.Tests.IntegrationTests.Utils;
+
+using Microsoft.AspNetCore.Mvc.Testing;
+
+using Moq;
+
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+using Xunit;
+
+namespace Altinn.Profile.Tests.IntegrationTests;
+
+public class TelemetryTests(WebApplicationFactory<Program> factory) 
+    : IClassFixture<WebApplicationFactory<Program>>, IDisposable
+{
+    private readonly WebApplicationFactorySetup<Program> _webApplicationFactorySetup = new(factory);
+
+    private MeterProvider _meterProvider;
+
+    [Fact]
+    public async Task Init_Verify_Creation_of_Meters()
```

Comment:
I thought we had a naming convention as `MethodName_Scenario_ExpectedBehavior`, [as described here](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests)

Reply:
Changed the name